packages/gatsby-source-filesystem/src/__tests__/create-file-node.js

```diff
@@ -59,77 +151,74 @@ describe(`create-file-node`, () => {
-            node[key] = node[key].replace(new RegExp(dname, `g`), `<DIR>`)
-            node[key] = node[key].replace(new RegExp(fname, `g`), `<FILE>`)
-          }
-        })
-        Object.keys(node.internal).forEach(key => {
-          if (typeof node.internal[key] === `string`) {
-            node.internal[key] = node.internal[key].replace(
-              new RegExp(dname, `g`),
-              `<DIR>`
-            )
-            node.internal[key] = node.internal[key].replace(
-              new RegExp(fname, `g`),
-              `<FILE>`
-            )
-          }
-        })
-
-        // Note: this snapshot should update if the mock above is changed
-        expect(node).toMatchInlineSnapshot(`
-          Object {
-            "absolutePath": "<DIR>/f",
-            "accessTime": "1970-01-01T00:02:03.456Z",
-            "atime": "1970-01-01T00:02:03.456Z",
-            "atimeMs": 123456,
-            "base": "f",
-            "birthTime": "1970-01-01T00:02:03.456Z",
-            "birthtime": "1970-01-01T00:02:03.456Z",
-            "birthtimeMs": 123456,
-            "blksize": 123456,
-            "blocks": 123456,
-            "changeTime": "1970-01-01T00:02:03.456Z",
-            "children": Array [],
-            "ctime": "1970-01-01T00:02:03.456Z",
-            "ctimeMs": 123456,
-            "dev": 123456,
-            "dir": "<DIR>",
-            "ext": "",
-            "extension": "",
-            "id": "uuid-from-gatsby",
-            "ino": 123456,
-            "internal": Object {
-              "contentDigest": "8d777f385d3dfec8815d20f7496026dc",
-              "description": "File \\"<DIR>/f\\"",
-              "mediaType": "application/octet-stream",
-              "type": "File",
-            },
-            "mode": 123456,
-            "modifiedTime": "1970-01-01T00:02:03.456Z",
-            "mtime": "1970-01-01T00:02:03.456Z",
-            "mtimeMs": 123456,
-            "name": "f",
-            "nlink": 123456,
-            "parent": null,
-            "prettySize": "123 kB",
-            "rdev": 123456,
-            "relativeDirectory": "<DIR>",
-            "relativePath": "<DIR>/f",
-            "root": "",
-            "size": 123456,
-            "sourceInstanceName": "__PROGRAMMATIC__",
-            "uid": 123456,
-          }
-        `)
+        const createNodeId = createMockCreateNodeId()
+
+        const emptyCache = {
+          get: jest.fn(),
+          set: jest.fn(),
+          directory: __dirname,
+        }
+
+        const node = await createFileNode(fname, createNodeId, {}, emptyCache)
+
+        testNode(node, dname, fname, fileHash)
+      } finally {
+        fs.unlinkSync(fname)
+      }
+    } finally {
+      fs.rmdirSync(dname)
+    }
+  })
+
+  it(`records the shape of the node from cache`, async () => {
+    const dname = fs.mkdtempSync(`gatsby-create-file-node-test`).trim()
+    try {
+      const fname = path.join(dname, `f`)
+      console.log(dname, fname)
```

Comment:
Stray console.log

Reply:
It was there already so I left it in but I've removed them now